src/constants/queryParams.ts

```diff
@@ -1,25 +1,26 @@
 export const QUERY_PARAMS = {
   // Core
-  query_string: "q",
-  exact_match: "e",
+  active_continuation_token: "act",
   category: "c",
-  region: "r",
+  continuation_tokens: "cts",
   country: "l",
-  year_range: "y",
+  exact_match: "e",
+  offset: "o",
+  query_string: "q",
+  region: "r",
   sort_field: "sf",
   sort_order: "so",
-  offset: "o",
-  active_continuation_token: "act",
-  continuation_tokens: "cts",
+  sort_within_page: "sp",
```

Comment:
Question: this sort option is more of a passage level sort than explicitly a sort within a particular page? Would `passage_sort`, or `sort_passages` (keeps as sp) would read easier?

Reply:
I had tried to keep this the same as the API query param, but you're right, it's not intuitive. I've gone with `passages_by_position` as I'd like to keep it a boolean.